### DISCUSSION TITLE
[CH] Aim to release  CH backend at 1.3

### DISCUSSION THREAD

**Discussion by baibaichen** [apache/gluten#7021] [General]
Hi All

CH backend 错过了 1.2 的发版，沟通下来，最好是在 1.3 （第四季度）正式 Release CH Backend 的第一版，这里是我想到的功能和需要做的事情

1. 完全支持 TPCH 和 TPCDS（3.3.1 和 3.5.1）
2. https://github.com/apache/incubator-gluten/issues/7028
3. https://github.com/apache/incubator-gluten/issues/6067
4. 性能优化，已知
   1. https://github.com/apache/incubator-gluten/issues/6975
5. 文档：
   1. 已支持的函数列表（Spark  3.5.1）https://github.com/apache/incubator-gluten/issues/6817
   2. 性能测试报告
      1. TPCH
      2. TPCDS
   3. Quick start

**Comment by zhanglistar** [comment]
This plan is totally enough for our first release on CH backend.
IMO, it's important to completely support TCP-H and TPC-DS, and the performance comparation result on scale factor = 1000. Need to figure it out how much work here.

**Reply by zhanglistar** [reply]
And supportting spark 3.5 is also important. We need do some work here.

**Comment by loudongfeng** [comment]
Shall we add this one too?  shade depency jars #7023
